c/utils.c

```diff
@@ -2169,6 +2169,29 @@ static int incrementPlaceValues(int *placeValues,
 }
 
 
+bool stringIsDigit(const char * str){
+  bool returnValue = TRUE;
+  for (int i = 0; i < strlen(str); i ++) {
+    if (!isdigit(str[i])) {
+      returnValue = FALSE;
+      break;
+    }
+  }
+  return returnValue;
+}
+
+#ifndef METTLE
```

Comment:
Why LE only?

Reply:
I can eliminate, but code is not tested for METTLE.

Reply:
My code change includes <limits.h> for LE and <metal/limits.h> for METAL.  I cannot include just one file for both LE and METAL.

Reply:
I understand. You don't have to include just one. Just make sure that Metal C still compiles at least.